tox.ini

```diff
@@ -40,6 +46,7 @@ skip_install = true
 commands = pip check
 
 [testenv:genreqs]
+basepython = python3.7
```

Comment:
what's the reason?

Reply:
The requirements.txt generated in python3.7 and python3.8 is different because some of our libraries use requirements filter like (python_version < 3.8).

Since production and circle-ci run python3.7, we must enforce requirements generation with 3.7

Reply:
Make sense!